### DISCUSSION TITLE
New feed is brutal

### DISCUSSION THREAD

**Discussion by stobias123** [community/community#66221] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Just got hit with the new feed update. 

I want it to look less like facebook, not more. I dont want changelog notes, pull request merge details, or "stuff you think I might be interested in".

I liked it much better when it was a small, unobtrusive list of activity - not a card per item.

**Comment by sureshg** [comment]
Exactly, completely agree with @stobias123  ...i used to find new repos from the activities of folks i am following. Can't see any of those details now (even after applying the filtering). Is there any way to make this optional?
